src/controllers/users_controller.js

```diff
@@ -81,4 +81,28 @@ module.exports.unlikesRecipe = ({body: {recipeId}, headers: {token}}, res) => {
         },
         (email, client, collection, next) => usersModel.unlikesRecipe(client, collection, email, recipeId, next)
     ], (err) => res.status(err ? 500 : 200).json(err ? err : undefined));
-};
\ No newline at end of file
+};
+
+module.exports.addAllergy = ({body: {allergy}, headers: {token}}, res) => {
+    async.waterfall([
+        (next) => usersModel.verifyToken(token, next),
+        ({email}, next) => {
+            usersModel.connect((err, client, collection) => {
+                next(err, email, client, collection)
+            })
+        },
+        (email, client, collection, next) => usersModel.addAllergy(client, collection, email, allergy, next)
+    ], (err) => res.status(err ? 500 : 200).json(err ? err : undefined))
+};
+
+module.exports.removeAllergy = ({body: {allergy}, headers: {token}}, res) => {
+    async.waterfall([
+        (next) => usersModel.verifyToken(token, next),
+        ({email}, next) => {
+            usersModel.connect((err, client, collection) => {
+                next(err, email, client, collection)
+            })
+        },
+        (email, client, collection, next) => usersModel.removeAllergy(client, collection, email, allergy, next)
+    ], (err) => res.status(err ? 500 : 200).json(err ? err : undefined))
+}
```

Comment:
add blank line at end of file

Reply:
I'll add it, but we need to make these a part of the lint rules, otherwise, it's very tedious to check manually.